site/static/examples/jobs/sample-paddlejob.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kubeflow.org/v1
+kind: PaddleJob
+metadata:
+  name: paddle-simple-cpu
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  paddleReplicaSpecs:
```

Comment:
```suggestion
spec:
  runPolicy:
    suspend: true
  paddleReplicaSpecs:
```
nit

Reply:
Previously, we decided not to specify the `suspend=true` and documented it as optional.

https://github.com/kubernetes-sigs/kueue/pull/778#discussion_r1195874307